lib/ex.coffee

```diff
@@ -106,16 +107,30 @@ class Ex
   tabp: => @tabprevious()
 
   edit: (range, filePath) ->
-    filePath = fs.normalize(filePath.trim())
+    filePath = filePath.trim()
+    if filePath[0] is '!'
+      force = true
+      filePath = filePath[1..].trim()
+    else
+      force = false
```

Comment:
This warrants some factoring:
- make a generic function that takes args and returns a tuple of [args, force], used with destructuring assignment: `[args, force] = extractForce(args)`.
- reuse in e.g. `:write` and `:quit`